README.md

```diff
@@ -149,6 +149,14 @@ var withDots = qs.parse('a.b=c', { allowDots: true });
 assert.deepEqual(withDots, { a: { b: 'c' } });
 ```
 
+Option `decodeDotInKeys` can be used to decode dot notations in keys
+Note: it only works with `allowDots` so you have to provide both options to be `true`:
+
+```javascript
+var withDots = qs.parse('name%252Eobj.first=John&name%252Eobj.last=Doe', { allowDots: true, decodeDotInKeys: true });
+assert.deepEqual(withDots, { 'name.obj': { first: 'John', last: 'Doe' }});
+```
```

Comment:
now that i think about it, do we need a parse option here? i feel like it should just be automatically decoded. what tests break if we default this option to `true`?